### PR TITLE
Add footer details

### LINKED ISSUE #16: [0.5pt] Add footer - group task
Create footer according to the wireframe: 

Screenshot 2021-06-27 at 21 42 29

### PR DESCRIPTION
**Pull request by manq2010**
Add footer
---

In this PR, the following was done:

- [x] Added a footer

### CODE REVIEW

**Review by wassimchakib — CHANGES_REQUESTED** [review]
Hi @manq2010 , 

Good job so far!
There are some issues that you still need to work on to go to the next step but you are almost there!

invalid-1


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Used correct Gitflow ✔️ 

## Required Changes ♻️
Please Try to style your header. 
 Expected : 123557332-ac3eee80-d790-11eb-91c8-9f8a6a38a4ff

Yours: 
image


- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 1]
```diff
@@ -16,5 +16,10 @@
             </ul>
         </nav>
     </header>
+
+    <footer class="f-container">
+        <p class="footer-details">Created by Microverse under CC lincense</p>
+    </footer>
+
```
Comment (wassimchakib) [inline src/index.html:23]: Please try to add style for your footer: 
Your design : 
image

Expected design : 


123557345-c082eb80-d790-11eb-969a-2fc82bbec349

**Review by wassimchakib — CHANGES_REQUESTED** [review]

**Review by wassimchakib — CHANGES_REQUESTED** [review]
## Required Changes ♻️
 Good job so far, 
download

- [ ] Please kindly follow Microverse's layout design requirements in the header section : 

Your layout : image

Expected Design : 
123557332-ac3eee80-d790-11eb-91c8-9f8a6a38a4ff


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Comment by manq2010** [comment]
Noted @wassimchakib, I have made the changes as requested!

image

**Review by wassimchakib — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @manq2010 , 

Valid-2



Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**